recipe/meta.yaml

```diff
@@ -10,50 +10,165 @@ source:
+        - python >={{ python_min }}
+        - asgiref >=3.4
+        - httpx >=0.23
+        - inflection >=0.3.1
+        - jsonschema >=4.17.3
+        - jinja2 >=3.0.0
+        - python-multipart >=0.0.15
+        - pyyaml >=5.1
+        - requests >=2.27
+        - starlette >=0.35
+        - typing-extensions >=4.6.1
+        - werkzeug >=2.2.1
+      run_constrained:
+        - a2wsgi >=1.7
+        - flask >=2.2
+        - swagger-ui-bundle >=1.1.0
+        - uvicorn >=0.17.6
+        - jsf >=0.10.0
+
+    test:
+      imports:
+        - connexion
+        - connexion.apps
+        - connexion.decorators
+      commands:
+        - connexion --version
+        - pip check
+      requires:
+        - pip
+        - python {{ python_min }}
+
+  - name: connexion
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+      run:
+        - python >={{ python_min }}
+        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
+    test:
+      requires:
+        - pip
+        - python {{ python_min }}
+      imports:
+        - connexion
+      commands:
+        - pip check
+
+  - name: connexion-flask
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+        - a2wsgi
+        - flask >=2.2.0
+        # - flask-asysnc >=2.2.0  # extra not provided
+      run:
+        - python >={{ python_min }}
+        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
+    test:
+      requires:
+        - pip
+        - python {{ python_min }}
+      imports:
+        - connexion
+        # see https://github.com/spec-first/connexion/blob/main/connexion/__init__.py
+        - connexion.apps.flask.FlaskApp
+        - connexion.App
+      commands:
+        - pip check
+
+  - name: connexion-swagger-ui
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+        - swagger-ui-bundle >=1.1.0
+      run:
+        - python >={{ python_min }}
+        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
+    test:
+      requires:
+        - pip
+        - python {{ python_min }}
+      imports:
+        - connexion
+        # - connexion.AsyncApp
+        # - connextion.options.SwaggerUIOptions
+      commands:
+        - pip check
+
+  - name: connexion-uvicorn
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+        - uvicorn >=0.34.0
+        - uvicorn-standard >=0.34.0
+      run:
+        - python >={{ python_min }}
+        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
+    test:
+      requires:
+        - pip
+        - python {{ python_min }}
+      imports:
+        - connexion
+        # - connexion.uvicorn
+      commands:
+        - pip check
+
+  - name: connexion-mock
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+        - jsf >=0.11.2
```

Comment:
Similarly here, this should be the same constraint as upstream:
https://github.com/spec-first/connexion/blob/3.2.0/pyproject.toml#L64